Objects/exceptions.c

```diff
@@ -207,20 +207,19 @@ BaseException_add_note(PyObject *self, PyObject *note)
         return NULL;
     }
 
-    if (!PyObject_HasAttr(self, &_Py_ID(__notes__))) {
-        PyObject *new_notes = PyList_New(0);
-        if (new_notes == NULL) {
+    PyObject *notes;
+    if (_PyObject_LookupAttr(self, &_Py_ID(__notes__), &notes) < 0) {
+        return NULL;
+    }
+    if (!notes) {
+        notes = PyList_New(0);
+        if (notes == NULL) {
```

Comment:
Thank you @serhiy-storchaka .   Shall we be consistent about ``!notes`` vs ``notes == NULL``?

Reply:
It is a matter of personal taste. I did not noticed that existing code used `== NULL` in near place. Of course, consistency matters! Thank you for noticing this.